.github/workflows/release.yml

```diff
@@ -0,0 +1,27 @@
+name: release assets
+on:
+  push:
+    tags:
+      - v*
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/setup-go@v1
+      with:
+        go-version: '1.13'
+    - uses: actions/checkout@v1
+    - name: run unit tests
+      run: make test-unit
+    - name: run system tests
+      run: make test-system
+    - name: create release assets
+      run: make create-release-assets
+    - name: upload release assets
+      uses: svenstaro/upload-release-action@1.1.0
```

Comment:
Maybe use https://github.com/actions/upload-release-asset?

Reply:
Unfortunately, it only supports uploading a single asset at a time, and doesn't support file patterns or globs when selecting the assets to upload. I started by trying to use it, then shifted over to this one.